src/apis/tests/test_scheduler.py

```diff
@@ -0,0 +1,88 @@
+from ..scheduler import cat_swarm
+
+import random
+import json
+import os
+f=open("testing_subset.json","r")
+data = f.read()
+input_data=json.loads(data)
+input_profs=input_data["users"]
+input_courses=input_data["courses"]
+input_classrooms=input_data["classrooms"]
+f.close()
+f=open("time_blocks.json","r")
+data = f.read()
+input_timeblocks = json.loads(data)
+def test_cso():
+    best_solution = cat_swarm.cat_swarm_optimization(input_profs, input_courses, input_classrooms, input_timeblocks,50, 1000)
```

Comment:
lower number of iterations to <=100, 1000 will make the test take way too long to run